ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/DataHubPlugin.groovy

```diff
@@ -108,6 +111,8 @@ class DataHubPlugin implements Plugin<Project> {
         project.task("hubSaveIndexes", group: scaffoldGroup, type: SaveIndexes,
             description: "Saves the indexes defined in {entity-name}.entity.json file to staging and final entity config in src/main/entity-config/databases directory")
 
+        project.tasks.mlPostDeploy.getDependsOn().add("hubGenerateExplorerOptions")
```

Comment:
Nice - though try this out in a project that doesn't have any dependsOn on mlPostDeploy, just to verify that there's no null-pointer here.